Rocket.Chat/Views/Cells/Chat/ChatMessageAttachmentView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ChatMessageAttachmentView.swift
+//  Rocket.Chat
+//
+//  Created by Luca Justin Zimmermann on 01/02/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+class ChatMessageAttachmentView: UIView {
+    class var defaultHeight: CGFloat {
+        return 0
+    }
+
+    static func heightFor(withText description: String?) -> CGFloat {
+        guard let text = description, text != "" else {
```

Comment:
You can use `!text.isEmpty` also, instead of checking for `empty` string. It will prevent from allocating another String in memory.